bin/arnold

```diff
@@ -37,6 +47,8 @@ declare -r CUSTOMER_DIR="${ARNOLD_PROJECT_PATH}/${RELATIVE_CUSTOMER_DIR}"
 declare -r VAULT_PASSWORD_FILE_NAME="password.gpg"
 
 # Binaries
+declare -r CURL="docker run --rm curlimages/curl"
```

Comment:
No please, don't use docker to run `curl`. Use directly `curl` or `wget`, `wget` is installed everywhere.

Reply:
> wget is installed everywhere.

That's a strong statement. I'm not sure to share this thought.

What about trying to use locally installed tools and fall back to the docker image?

Reply:
Ok then. After an IRL discussion, `curl` is now a project dependency. Arnold will fail with an explicit error message if it's not installed.